webapp/components/search_results_item.jsx

```diff
@@ -36,6 +36,14 @@ export default class SearchResultsItem extends React.Component {
         $('.sidebar--right').removeClass('move--left');
     }
 
+    shouldComponentUpdate(nextProps) {
```

Comment:
Can you remove this function? As it is it will cause us to miss renders that we should be doing and it's not really needed in this context